hysds/__init__.py

```diff
@@ -2,6 +2,6 @@
 from __future__ import print_function
 from __future__ import division
 from __future__ import absolute_import
-__version__ = "0.3.5"
+__version__ = "0.3.6"
```

Comment:
@DustinKLo: let's shoot for version `0.4.0` instead and leave the `0.3.*` for the ES 1.7 branches.